README.md

```diff
@@ -50,3 +50,86 @@ To run over all the files in the repo manually:
 To run pre-commit automatically before committing in a local repo, install git hooks:
 
     pre-commit install
+
+
+# Tracing
+
+If you're wondering what tracing is and how to use this information, you most probably don't need tracing at this point.
+Despite this passive agressive tone (which is introduced to add a little bit of humor), we provide some information to get you started with tracing.
+
+## Tracepoints and source files
+
+Sunrise added the tracepoints within the `read` and `write` functions of the hardware interface.
```

Comment:
You mean with "Sunrise added..." that this has been merged upstream? Otherwise I'd just say "Tracepoints have been added to ...".
Otherwise you'll have to link the upstream.